projects/project-player-library/src/lib/pages/show-certificate/show-certificate.component.html

```diff
@@ -0,0 +1,28 @@
+
+<div class="">
+
+    <div class="flex items-center">
+        <div class="text-lg font-bold">{{ "CERTIFICATE" | translate }}</div>
+    </div>
+    <div *ngIf="projectDetails.certificate.eligible"  class="mt-4">
+        <div class="flex justify-between items-center">
+            <div>
+                <h4 >{{projectDetails?.title}}</h4>
+                <p *ngIf="projectDetails?.certificate?.issuedOn">{{ getFormattedDate(projectDetails?.certificate?.issuedOn) }}</p>
```

Comment:
Instead of calling function for date formatting , can we try to use pipes
Check how the date is formatted in task-card

Reply:
done